api/v1/credentialset_types_test.go

```diff
@@ -53,12 +54,11 @@ func TestCredentialSetSpec_ToPorterDocument(t *testing.T) {
 				Credentials:   tt.fields.Credentials,
 			}
 			got, err := cs.ToPorterDocument()
-			if (err != nil) != tt.wantErr {
-				t.Errorf("CredentialSetSpec.ToPorterDocument() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("CredentialSetSpec.ToPorterDocument() = \n%v, want \n%v", string(got), string(tt.want))
+			if tt.wantErrMsg == "" {
```

Comment:
It looks like you are missing an else statement here to handle when wantErrMsg is not empty?